src/postgres_mcp/dta/__init__.py

```diff
@@ -1 +1,23 @@
-"""PostgreSQL Database Tuning Advisor (DTA) tool."""
+from .dta_calc import DatabaseTuningAdvisor
```

Comment:
Do we need an `__init__.py`. Should we we using namespace packages when possible?